qi-doc/scribblings/list-operations.scrbl

```diff
@@ -0,0 +1,76 @@
+#lang scribble/doc
+@require[scribble/manual
+	 (for-label racket/list
+	 	    racket/base)]
+
+@title{List Operations}
+
+@defmodule[qi/list]
+
+This module defines bindings that can leverage stream fusion /
+deforestation optimization when found in succession within a
+flow. When not part of optimized flow, their behavior is identical to
+the bindings of the same name from @racketmodname[racket/base] and
+@racketmodname[racket/list].
```

Comment:
Suggestion: "This module defines bindings which are optimized when used together in a flow.
Their behavior is otherwise identical to bindings of the same name from […]"

Specifics about the optimizations can be deferred to a later section.

Reply:
Postponed to a separate PR for improving documentation.